src/index.ts

```diff
@@ -1,16 +1,16 @@
-import { IncomingMessage, ServerResponse } from 'http'
-import { ParsedUrlQuery } from 'querystring'
-import { serialize, SerializeOptions } from '@tinyhttp/cookie'
+import type { IncomingMessage, ServerResponse } from 'node:http'
+import type { ParsedUrlQuery } from 'node:querystring'
+import { type SerializeOptions, serialize } from '@tinyhttp/cookie'
 import { sign } from '@tinyhttp/cookie-signature'
 import { Tokens } from './token'
 
 export interface CSRFRequest extends IncomingMessage {
   csrfToken(): string
   secret?: string | string[]
-  signedCookies?: any
-  cookies?: any
+  signedCookies?: unknown
```

Comment:
shouldn't these be a Record type?

Reply:
I put an `assertRecord()` function below to make sure it's `Record` type

Reply:
yeah but why not type them already as records?

Reply:
I forgot why I didn't do that.

Reply:
could you update it then? i don't think we need a runtime assertion

Reply:
Sure will do that tomorrow